src/common/box_filters.c

```diff
@@ -139,27 +141,31 @@ static void blur_vertical_1ch_sse(float *buf, const int height, const int width,
 #endif /* __SSE2__ */
 
 // invoked inside an OpenMP parallel for, so no need to parallelize
-static void blur_vertical_4wide(float *buf, const int height, const int width, const int radius, float *scanline)
+/*static*/ void blur_vertical_4wide(float *const restrict buf, const int height, const int width, const int radius,
+                                    float *const restrict scanline)
 {
 #ifdef __SSE2__
-  if (darktable.codepath.SSE2)
+  if (darktable.codepath.SSE2 & 0)
```

Comment:
@ralfbrown :
I see that now, was this expected ? Looks like debug code to disable SSE path ?

Reply:
I see, this was cleaned up in next commit, that's why I didn't see it.

Reply:
Yep, debug code.